BootloaderCorePkg/Tools/PrepareBuildComponentBin.py

```diff
@@ -180,7 +193,7 @@ def BuildFspBins (fsp_dir, sbl_dir, platform, flag):
     print ('Done\n')
 
     print ('Applying QEMU FSP patch ...')
-    patch_dir = os.path.join(sbl_dir, 'Silicon/QemuSocPkg/FspBin/Patches')
+    patch_dir = os.path.join(os.environ['PLT_SOURCE'], 'Silicon/QemuSocPkg/FspBin/Patches')
```

Comment:
Does this work ?   If we set PLT_SOURCE to a valid path, you won't be able to locate QEMU patches since it is always in the SblOpen.

Reply:
it works if build QEMU @ SblOpen folder using buildLoader.py.

Reply:
ok